test/cpp/jit/test_lite_interpreter.cpp

```diff
@@ -300,6 +302,173 @@ void testLiteInterpreterBuiltinFunction() {
+  Module a("A");
+  a.define(R"JIT(
+    def forward(self, x):
+      return 2 * x + 5
+  )JIT");
+  Module b("B");
+  b.register_module("A0", a);
+  b.define(R"JIT(
+    def forward(self, x):
+      return self.A0.forward(x) + 1
+  )JIT");
+
+  std::stringstream ss;
+  b._save_for_mobile(ss, {}, true);
+  mobile::Module bc = _load_for_mobile(ss);
+
+  std::unordered_set<std::string> module_debug_info_set;
+  size_t pc = 0;
+  while (true) {
+    std::string module_info = bc.get_forward_method_debug_info(pc);
+    if (module_info == "<no module info for pc>") {
+      break;
+    }
+    if (!module_info.empty()) {
+      module_debug_info_set.insert(module_info);
+    }
+    ++pc;
+  }
+
+  std::unordered_set<std::string> expected_result(
+      {"top(B).forward", "top(B).A0(A).forward"});
+  AT_ASSERT(module_debug_info_set == expected_result);
+}
+
+void testLiteInterpreterTwoSubmodulesInfo() {
+  Module a("A");
+  a.define(R"JIT(
+    def forward(self, x):
+      return x + 1
+  )JIT");
+  Module b("B");
+  b.define(R"JIT(
+    def forward(self, x):
+      return x + 2
+  )JIT");
+  Module c("C");
+  c.register_module("A0", a);
+  c.register_module("B0", b);
+  c.define(R"JIT(
+    def forward(self, x):
+      return self.A0.forward(x) + self.B0.forward(x)
+  )JIT");
+
+  std::stringstream ss;
+  c._save_for_mobile(ss, {}, true);
+  mobile::Module bc = _load_for_mobile(ss);
+
+  std::unordered_set<std::string> module_debug_info_set;
+  size_t pc = 0;
+  while (true) {
+    std::string module_info = bc.get_forward_method_debug_info(pc);
+    if (module_info == "<no module info for pc>") {
+      break;
+    }
+    if (!module_info.empty()) {
+      module_debug_info_set.insert(module_info);
+    }
+    ++pc;
+  }
+
+  std::unordered_set<std::string> expected_result(
+      {"top(C).forward", "top(C).A0(A).forward", "top(C).B0(B).forward"});
+  AT_ASSERT(module_debug_info_set == expected_result);
+}
+
+void testLiteInterpreterSequentialModuleInfo() {
+  Module a("A");
+  a.define(R"JIT(
+    def forward(self, x):
+      return x + 1
+  )JIT");
+  Module b("B");
+  b.define(R"JIT(
+    def forward(self, x):
+      return x + 2
+  )JIT");
+  Module c("C");
+  c.register_module("A0", a);
+  c.register_module("B0", b);
+  c.define(R"JIT(
+    def forward(self, x):
+      return self.A0.forward(self.B0.forward(x))
+  )JIT");
+
+  std::stringstream ss;
+  c._save_for_mobile(ss, {}, true);
+  mobile::Module bc = _load_for_mobile(ss);
+
+  std::unordered_set<std::string> module_debug_info_set;
+  size_t pc = 0;
+  while (true) {
+    std::string module_info = bc.get_forward_method_debug_info(pc);
+    if (module_info == "<no module info for pc>") {
+      break;
+    }
+    if (!module_info.empty()) {
+      module_debug_info_set.insert(module_info);
+    }
+    ++pc;
+  }
+
+  std::unordered_set<std::string> expected_result(
+      {"top(C).A0(A).forward", "top(C).B0(B).forward"});
+  AT_ASSERT(module_debug_info_set == expected_result);
```

Comment:
Add 
- Where we get top(C).A0(A).forward.B0(b).forward.
- And also one where you have A0 and A1 and then you can get only type information.